helper/restore_helper.go

```diff
@@ -47,8 +48,33 @@ var (
 type RestoreReader struct {
 	bufReader  *bufio.Reader
 	seekReader io.ReadSeeker
+	pluginCmd  *pluginCmd
 	readerType ReaderType
-	errBuf     bytes.Buffer
+}
+
+// Log plugin's stderr as warnings.
+func (r *RestoreReader) logPlugin() {
+	if r.pluginCmd != nil {
+		errLog := strings.Trim(r.pluginCmd.stderrBuffer.String(), "\x00")
+		if len(errLog) != 0 {
+			gplog.Warn(fmt.Sprintf("Plugin log: %s", errLog))
```

Comment:
I think it is better to make a wrapper `logWarn` like the existing `logError` and `log`. And add segment identification there as well.